testsuite/integration/microprofile-tck/jwt/pom.xml

```diff
@@ -182,6 +182,113 @@
             </build>
         </profile>
 
+        <!-- Test against bootable jar -->
+        <profile>
+            <id>bootablejar.profile</id>
+            <activation>
+                <property>
+                    <name>ts.bootable</name>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.wildfly.arquillian</groupId>
+                    <artifactId>wildfly-arquillian-container-bootable</artifactId>
+                    <scope>test</scope>
+                </dependency>
```

Comment:
Is this dependency defined in parent pom?

Reply:
Yep, here it is: https://github.com/fabiobrz/wildfly/blob/adjust-ts-for-bootable-jar/pom.xml#L8644